src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3434,14 +3435,14 @@ public override SyntaxNode DefaultExpression(ITypeSymbol type)
             return DefaultExpression(type.GenerateTypeSyntax());
         }
 
-        internal override SyntaxNode AddParentheses(SyntaxNode expression)
+        internal override SyntaxNode AddParentheses(SyntaxNode expression, bool includeElasticTrivia = true, bool addSimplifierAnnotation = true)
```

Comment:
New optional parameters added to this internal method were already optional parameters for `ExpressionSyntaxExtensions.Parenthesize` extension method.